### ISSUE TITLE
[MU4 Issue] Audio FX plugins in Mixer should have option to move the order

### ISSUE THREAD

**Issue by jessjwilliamson** [musescore/MuseScore#9754]
I've shown the current situation with Musescore then expected (similar) behaviour in Logic Pro.
It is DAW standard that a user is able to click and drag the ordering of a plugin. A user would want this functionality, usually to do with post audio processing reasons.  For example, a user applies an EQ plugin before a compression plugin then later decides to swap around for the sake of audio processing sound (this is more about mixing). If not mentioned - top plugins are processed first then downwards are post processing. 
In a nutshell - Plugins should be able to be dragged to other slots like Logic can do. 

https://user-images.githubusercontent.com/71656189/142038013-0e47a766-bf86-4e95-83dc-bf7c867dc06a.mov

**Desktop (please complete the following information):**
 - OS: Windows and Mac

**Comment by Tantacrul** [comment]
This is legit but I think I'm going to bump it to 4.x.
